cluster-provision/k8s/1.28/provision.sh

```diff
@@ -73,7 +73,13 @@ function pull_container_retry() {
 }
 
 # Install modules of the initrd kernel
-dnf install -y "kernel-modules-$(uname -r)"
+
+curl -L https://composes.stream.centos.org/production/CentOS-Stream-9-20240126.0/compose/BaseOS/x86_64/os/Packages/kernel-modules-5.14.0-411.el9.x86_64.rpm -o ./kernel-modules-5.14.0-411.el9.x86_64.rpm 
```

Comment:
I think we need modules for 5.14.0.-**419**
It seem the vargrantbox image of version 0240212.0 runs kernel 5.14.0-419.el9.x86_64.
I saw the following message in the image build log
```
14:47:21,627 NOTICE kernel:Linux version 5.14.0-419.el9.x86_64 (mockbuild@x86-05.stream.rdu2.redhat.com) (gcc (GCC) 11.4.1 20231218 (Red Hat 11.4.1-3), GNU ld version 2.35.2-42.el9) #1 SMP PREEMPT_DYNAMIC Wed Feb 7 23:01:41 UTC 2024
```
https://kojihub.stream.centos.org/kojifiles/packages/CentOS-Stream-Vagrant/9/20240212.0/data/logs/image/oz-x86_64.log